packages/amplify-cli/src/init-steps/s9-onSuccess.js

```diff
@@ -141,6 +153,31 @@ function generateGitIgnoreFile(context) {
   }
 }
 
+function replaceTags(arr) {
+  let returnedArr = arr;
+
+  returnedArr.forEach(tagObj => {
+    if (
+      tagObj['Value'].includes('{project-env}') ||
+      tagObj['Value'].includes('{project-name') ||
+      tagObj['Value'].includes('{cli-version')
+    ) {
+      const replaceWith = {
+        '{project-name}': 'foobar',
```

Comment:
This function isn't being used and was initially added for testing. I forgot to delete this part of the code before pushing, but that will be taken care of on the next commit pushed.